content-src/components/LinkMenu/LinkMenu.js

```diff
@@ -12,13 +12,15 @@ const LinkMenu = React.createClass({
     };
   },
   userEvent(event) {
-    const {page, source, index, dispatch} = this.props;
+    const {page, source, index, dispatch, site} = this.props;
     if (page && source) {
       let payload = {
         event,
         page: page,
         source: source,
-        action_position: index
+        action_position: index,
+        url: site.recommended ? site.url : null,
+        recommender_type: site.recommended ? site.recommender_type : null
```

Comment:
actually let me double check with kate, i don't want to to add a condition that will never be met and then break all the metrics